doc/_static/js/gtm-insert.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2023 Nordic Semiconductor ASA
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
nordic bsd 5 clause?

Reply:
This snippet I assume is from google though: https://developers.google.com/tag-platform/tag-manager/web
Which use apache-2

Reply:
ok, license check just annoying here then, will need merge superpowers

Reply:
I think they stem from all the conf.py's which are all missing. I can add one.

Reply:
I added `SPDX-License-Identifier: LicenseRef-Nordic-5-Clause` to all conf.py files, which seems to have worked, but the Apache-2.0 in this file looks to be not allowed. My knowledge on licenses are limited, can we use another license here?

Reply:
@carlescufi will need merge superpowers